app/src/main/res/menu/starred_list_menu.xml

```diff
@@ -36,5 +36,5 @@
             android:titleCondensed="@string/menu_item_title_delete_all"
             android:id="@+id/menu_item_delete_all_favorites"
             app:showAsAction="ifRoom"
-            android:icon="@drawable/ic_delete_white_24dp"/>
+            android:icon="@drawable/ic_delete"/>
 </menu>
```

Comment:
In assets, you can simplify the files names like:

- assets/bell_list.svg
- assets/healing_list.svg
- assets/star_list.svg